### ISSUE TITLE
Ruby 2.4 support

### ISSUE THREAD

**Issue by denislins** [grpc/grpc#9296]
Ruby 2.4.0 was released last December 25th, and among many changes, it unifies both `Fixnum` and `Bignum` into `Integer`.

Unfortunately, the C extension relies on the fixnum symbol, which was apparently removed, even though the Ruby class still exists, only deprecated.

When I start a grpc server using Ruby 2.4.0, I get this error:

```
/usr/local/bundle/gems/grpc-1.0.1-x86_64-linux/src/ruby/lib/grpc/grpc.rb:37:in `require_relative':
/usr/local/bundle/gems/grpc-1.0.1-x86_64-linux/src/ruby/lib/grpc/grpc_c.so: undefined symbol: rb_cFixnum (LoadError)
```

Are there plans to support this version of ruby in the short run?

**Comment by apolcyn** [comment]
@denislins the next release at v1.1 along with the updated protobuf dependency to 3.2, is planned to run on ruby 2.4.0.

**Comment by apolcyn** [comment]
1.1.2 release was just published with 2.4 support - unfortunately the 1.1.0 release was postponed.

note though that the latest version of google-protobuf at 3.1 doesn't support it though, but when a new release with support is published it can be picked up and used.

**Comment by blowmage** [comment]
I have been able to use Ruby 2.4.0 with grpc 1.1.2 and google-protobug 3.2.0. Can this issue be closed?
